packages/terra-data-grid/tests/jest/Cell.test.jsx

```diff
@@ -109,7 +109,7 @@ describe('Cell', () => {
     expect(blankCell).toHaveLength(1);
 
     const cellContent = blankCell.find('div');
-    expect(cellContent.text()).toBe('Terra.worklistDataGrid.blank');
+    expect(cellContent.text()).toBe('Terra.data-grid.blank');
```

Comment:
- I noticed that here we switched from camelCase to snake-case naming in translation variables. I checked with some other components and it looks like for consistancy it's better to keep it lowCamelCase (`Terra.dataGrid.blank` in this case) through all translations.

Reply:
Updated